libs/utils/src/http/json.rs

```diff
@@ -25,8 +25,12 @@ pub async fn json_request_or_empty_body<T: for<'de> Deserialize<'de>>(
     if body.remaining() == 0 {
         return Ok(None);
     }
-    serde_json::from_reader(body.reader())
-        .context("Failed to parse json request")
+
+    let mut deser = serde_json::de::Deserializer::from_reader(body.reader());
+
+    serde_path_to_error::deserialize(&mut deser)
+        // intentionally stringify because the debug version is not helpful in python logs
+        .map_err(|e| anyhow::anyhow!("Failed to parse json request: {e}"))
```

Comment:
this will also send this error to the http client, right? I am generally a bit wary when it comes to exposing too chatty error messages to potentially unprivileged clients, but these errors won't expose any private details, will they?

Reply:
No, the it's just `{}: {}` where on the left we have path to the node, on the right we have the original terse serde error. So it's based on only what the user sent, and even for that I don't think there should be any case where serde::Deserialize would error out with the input... Well, I guess there could be, but still, we don't have personally identifiable information, and this is 99.999% of time hit in tests.